### ISSUE TITLE
Confusion over the template email for incomplete samples

### ISSUE THREAD

**Issue by wasade** [qiime/qiime_web_app#558]
There is a substantial amount of confusion still over the template email for incomplete samples as to if the sample still needs to be logged or if there is missing consent

**Comment by teravest** [comment]
I agree this is confusing.  There is no way to log it if consent is missing, and right now the code is only checking to see if the info is missing it doesn't do any consent checking at all. Actually there isn't really a good way at all from wwwadmin to see if a sample has consent or not.  This is definitely something we should look into fixing.

**Comment by adamrp** [comment]
Even in the database, the only way we can tell if a sample has consent is
if there is a row in ag_human_survey for the participant with which the
sample is associated.  There are (at least) a couple problems with this:
- The participant can disassociate the sample with him/herself
- The participant can remove him/herself
